test/jasmine/tests/bar_test.js

```diff
@@ -674,6 +922,88 @@ describe('A bar plot', function() {
             done();
         });
     });
+
+    it('should coerce text-related attributes', function(done) {
```

Comment:
Nice test. Can you make sure that `Plotly.restyle` works with the new attributes too?